google-datacatalog-rdbms-connector/tests/google/datacatalog_connectors/rdbms/scrape/metadata_scraper_test.py

```diff
@@ -105,11 +104,9 @@ def test_scrape_metadata_with_credentials_overriding_base_metadata_query_should_
 
         self.assertEqual(1, len(schemas_metadata))
 
-    @mock.patch('{}.'.format(__SCRAPE_PACKAGE) +
-                'metadata_scraper.MetadataNormalizer.' +
+    @mock.patch('{}.'.format(__NORMALIZER_CLASS) +
```

Comment:
```suggestion
    @mock.patch('{}.get_exact_table_names_from_dataframe'.format(__NORMALIZER_CLASS)
```
I would like to use the same approach to patch methods from the normalizer class, so keeping the code consistent. If you agree, please apply this change and replace all similar occurrences in this file.

Reply:
makes sense!